sbi/src/tee_guest.rs

```diff
@@ -22,6 +22,18 @@ pub enum TeeGuestFunction {
         /// a1 = length of the region
         len: u64,
     },
+    /// Removes the specified range of guest physical address space from the emulated MMIO regions. Upon
```

Comment:
This breaks the backwards compatibility but given we are still in the development phase I thought this should be fine. This is to mainly keep `AddMmioRegion` and `RemoveMmioRegion` together.